14/umbraco-cms/fundamentals/design/templates/README.md

```diff
@@ -93,39 +142,36 @@ For instance, if you want to be able to add HTML to your `<head>` tags write:
 <html>
     <head>
         <title>Title</title>
-        @RenderSection("Head")
+        @RenderSection("SectionName", false)
     </head>
 
     <body>
     </body>
 </html>
 ```
 
-By default, when rendering a named section, the section is not mandatory. To make the section mandatory, add `true` or enable **Section is mandatory** field in the **Sections** option.
+You can decide whether a section should be mandatory or not. Making a section mandatory means that any template using the Master Template is required to have the section defined.
 
-```csharp
-@RenderSection("Head", true)
-```
+{% hint style="info" %}
+Keep in mind that whenever a mandatory named section is missing, it will result in errors on your website.
+{% endhint %}
 
-![Create partial](../../../../../10/umbraco-cms/fundamentals/design/templates/images/render-named-sections-v10.png)
+To make the section mandatory, you have two options:
 
-On your child page template call `@section Head {}` and then type your markup that will be pushed into the Master Template:
+* Check the **Section is mandatory** field when using the **Sections** dialog in the backoffice.
+* Add `true` to the code tag as shown in the example below.
```

Comment:
Is it possible to put the first point as the second along with the image? Currently, the image is slightly out of place after the code.

Reply:
How about I add the code snippet as part of the bullet? And then also change the order of the items in the list. That wont break up the list at least.

Reply:
I think that could work 💪